classes/log/store.php

```diff
@@ -167,6 +167,25 @@ public function get_max_batch_size() {
         return $this->get_config('maxbatchsize', 100);
     }
 
+    /**
+     * Take event data from the emit_task
+     * and add to the sent log if it doesn't exist already.
+     *
+     * @param array $events raw event data
+     * @return array
```

Comment:
There is no return, so we should remove this comment

Reply:
I have done this, but moved the code to the appropriate cron sub-task.